docs/development.md

```diff
@@ -1,32 +1,136 @@
+  <tbody valign="top"><tr>
+    <td>
+      <p><pre># apt install docker</pre></p>
+      <p>Start the Docker daemon in another terminal, and leave it running in the background:</p>
+      <p><pre>$ dockerd</pre></p>
+    </td>
+    <td>
+      <p><pre># apt install docker</pre></p>
+      <p>Install <a href="https://docs.docker.com/docker-for-windows/install-windows-home">Docker Desktop</a>, which you'll be using as the Docker daemon, and leave it running in the background.</p>
+      <p>If you have set <code>appendWindowsPath=false</code> in your WSL config, then you may hit an error along the lines of <code>"docker-credential-desktop.exe": executable file not found in $PATH</code>. In this case you should either add <code>/mnt/c/Program\ Files/Docker/Docker/resources/bin</code> to your PATH, or <a href="https://github.com/rossjrw/dotfiles/blob/3c5445abb138b735cc3caf61f070c9125fa87d2f/.profile#L28">create a symlink</a> in a directory that is in your PATH already.</p>
+    </td>
+  </tr></tbody>
+</table>
 
-You can create a docker container using the following:
+The first step is to build the Docker image, which is a single source of information for Docker to use in a container later:
 
 ```bash
-$ docker build . -t scpwiki/wikijump:local
-$ docker create --name wikijump -p 80:80 scpwiki/wikijump:local
-$ docker start wikijump
+$ cd installer
+$ docker build . -t scpwiki/wikijump:local --no-cache
 ```
 
-Instead of building Wikijump locally, you can also pull the image from the Docker Hub:
+Then load that image into a container, which is a VM that runs only the image you created:
 
 ```bash
-$ docker create --name wikijump -p 80:80 scpwiki/wikijump:latest
+$ docker create --name wj --publish 80:80 --publish 443:443 -it -d scpwiki/wikijump:local
 ```
 
-Then terminate it:
+The container is already running, but to start the services inside it that run Wikijump you will need to send a start command:
 
 ```bash
-$ docker stop wikijump
+$ docker start wj
 ```
 
-Alternatively, you can install to your local system using `install.sh`. This may require tinkering depending on your exact platform and environment.
+Then navigate to https://www.wikijump.test in your browser.
 
+-----
+
+To stop Wikijump:
+
+```bash
+$ docker stop wj
+```
+
+It's useful to keep track of existing Docker images and containers, and destroy them when you no longer need them, so you don't waste space rebuilding the same image over and over. If you are using Docker Desktop, you can manage containers and images from the GUI. Otherwise, on command line:
+
+```bash
+$ docker container ls  # List containers
+$ docker rm [ID]       # Destroy the container with this ID
+$ docker images        # List images
+$ docker rmi [ID]      # Remove the image with this ID
+```
+
+### Installation via Docker from the Docker Hub
+
+Instead of building Wikijump locally, you can also pull the `latest` image from the Docker Hub:
```

Comment:
```suggestion
Instead of building Wikijump locally, you can also pull the `latest` image from the Docker Hub:

(Note that this image may not be fully up-to-date with the current `master`)
```